docs/guides/use_checkpointing.md

```diff
@@ -1,20 +1,23 @@
 ---
-jupytext:
-  formats: ipynb,md:myst
-  text_representation:
-    extension: .md
-    format_name: myst
-    format_version: 0.13
-    jupytext_version: 1.13.8
-kernelspec:
-  display_name: Python 3 (ipykernel)
-  language: python
-  name: python3
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
 ---
 
-# Checkpointing with `flax.training.checkpoints`
+# Save and load checkpoints
 
-In this guide, you will learn about [`flax.training.checkpoints`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#module-flax.training.checkpoints)—a simplistic and generic checkpointing library built into Flax. With Flax Checkpoints, you can save and load model parameters, metadata, and a variety of Python data. In addition, it provides basic features for versioning, automatic bookkeeping of past checkpoints, and asynchronous saving to reduce training wait time.
+In this guide, you will learn about [`flax.training.checkpoints`](https://flax.readthedocs.io/en/latest/api_reference/flax.training.html#module-flax.training.checkpoints) — a simplistic and generic checkpointing library built into Flax. Flax allows you to save and load model parameters, metadata, and a variety of Python data using the library [Orbax](https://github.com/google/orbax), which provides a more customizable and flexible API for various array types and storage formats. In addition, Flax provides basic features for versioning, automatic bookkeeping of past checkpoints, and asynchronous saving to reduce training wait time.
```

Comment:
Since checkpointing will be taken care of by Orbax, maybe we can omit `flax.training.checkpoints` now.

For example:

```suggestion
In this guide, you will learn about saving and loading checkpoints with Flax and Orbax](https://github.com/google/orbax). Flax allows you to save and load model parameters, metadata, and a variety of Python data using Orbax, which provides a more customizable and flexible API for various array types and storage formats. In addition, Flax provides basic features for versioning, automatic bookkeeping of past checkpoints, and asynchronous saving to reduce training wait time.
```

Then, in the **Ongoing migration** note, we can add that:

```diff
**_Ongoing migration:_** In the foreseeable future, Flax's checkpointing
functionality will gradually be migrated to Orbax
+ from `flax.training,checkpoints`.

```

Does this make sense? LMKWYT :+1:

Reply:
Sounds good! Done.